metadata.go

```diff
@@ -80,12 +83,69 @@ var debianLicenseText = map[string]string{
 `,
 }
 
+var githubRegexp = regexp.MustCompile(`github\.com/([^/]+/[^/]+)`)
+
+func findGitHubRepo(gopkg string) (repo string, err error) {
+	if strings.HasPrefix(gopkg, "github.com/") {
+		repo = strings.TrimPrefix(gopkg, "github.com/")
+		return
+	}
+	resp, err := http.Get("https://" + gopkg + "?go-get=1")
+	if err != nil {
+		return
+	}
+	defer resp.Body.Close()
+	z := html.NewTokenizer(resp.Body)
```

Comment:
At first, I just randomly picked one of the methods.
I tried to use html.Parse after your suggestion, but seems it's not shorter. The most code is to parse meta content manually.
And I find if to walk through a node link, more code is needed to jump out when first github regexp matches.
So I'd like to use html.NewTokenizer.